### PR TITLE
Update redirect notice for UCL-RITS.

### PR DESCRIPTION
**Pull request by samcunliffe**
Thought we might want to be a bit more verbose here.

## Related to

- https://github.com/UCL-ARC/github-administration/issues/7

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-RITS is dead. Long live [ARC](https://github.com/UCL-ARC).
+Research IT Services (RITS) was the first team of research software engineers and research computing specialists in University College London.
+
+The team was moved to the [**Centre for Advanced Research Computing**](https://ucl.ac.uk/arc) ([@UCL-ARC](https://github.com/UCL-ARC)) in 2022.
```
Comment (samcunliffe) [inline profile/README.md:3]: 2022, right?

Comment (paddyroddy) [reply profile/README.md:3]: No 2021. When I started in November 2021 it was already ARC - although my card says RITS.

File: profile/README.md [hunk 2]
```diff
@@ -1 +1,9 @@
-RITS is dead. Long live [ARC](https://github.com/UCL-ARC).
+Research IT Services (RITS) was the first team of research software engineers and research computing specialists in University College London.
```
Comment (paddyroddy) [inline profile/README.md:1]: Just a thought, not pushing it
```suggestion
Research IT Services (RITS) was the first team of research technical professionals in University College London.
```

**Review by paddyroddy — APPROVED** [review]
Looks good

File: profile/README.md (hunk 2 above)
Comment (jonc125) [reply profile/README.md:1]: I think I'd stick with the more verbose description, as there were other technical teams with different specialisms in existence, just not centrally for the whole university with an RSE team too.

**Review by jonc125 — APPROVED** [review]

**Pull request merged**